blockchain/blockchain.go

```diff
@@ -290,17 +297,7 @@ func (bc *blockchain) Start(ctx context.Context) error {
 	if tipHeight == 0 {
 		return nil
 	}
-	if bcCtx, ok := protocol.GetBlockchainCtx(ctx); ok {
-		for _, p := range bcCtx.Registry.All() {
-			if s, ok := p.(lifecycle.Starter); ok {
-				if err := s.Start(ctx); err != nil {
-					return errors.Wrap(err, "failed to start protocol")
-				}
-			}
-		}
-	}
-
-	return bc.startExistingBlockchain(ctx)
```

Comment:
why do we still keep https://github.com/iotexproject/iotex-core/pull/1819/files#diff-646235a88d91f225fc74104cca24062aL513-R546 ?

Reply:
my fault,forgot to delete when merge